src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.File.cs

```diff
@@ -38,26 +38,13 @@ public ProjectRootElement ProjectRoot
             }
         }
 
-        public string Sdk
-        {
-            get
-            {
-                var sdk = ProjectRoot.Sdk;
-
-                if (sdk is null)
-                {
-                    throw new ArgumentOutOfRangeException("Should check IsSdk property first");
-                }
-
-                return sdk;
-            }
-        }
-
         public bool IsSdk =>
-            ProjectRoot.Sdk is not null && ProjectRoot.Sdk.Contains(MSBuildConstants.DefaultSDK, StringComparison.OrdinalIgnoreCase);
+            Sdk.Contains(MSBuildConstants.DefaultSDK) || Sdk.Contains(MSBuildConstants.WebSdk);
 
         public ICollection<string> Imports => new ImportsCollection(ProjectRoot);
 
+        public ICollection<string> Sdk => ProjectRoot.Sdk.Split(';').ToList();
```

Comment:
You should implement the ICollection against ProjectRoot so you can add/remove stuff